conf/wordforms_main.txt

```diff
@@ -1,11 +1,20 @@
+W > Weg
+Al > Allee
+Str > Strasse
+Pl > Platz
```

Comment:
Wordforms are not lower/upper case sensitive, that's only for orthographic reasons. (and it is also easier to look for german / french / italian words since I did not found anywhere how to insert comments in this kind of files)